### ISSUE TITLE
Fatal Python error: Aborted in spatial2channel.py line 77 when using 1536_cascade pipeline

### ISSUE THREAD

**Issue by xtianmir** [visualbruno/ComfyUI-Trellis2#113]
### Environment
- **GPU:** NVIDIA GeForce RTX 4090 24GB
- **PyTorch:** 2.8.0+cu128
- **Python:** 3.12.10
- **ComfyUI-Trellis2:** latest
- **OS:** Windows 11

### Description
When using the `Trellis2 - Mesh With Voxel Multi-View Generator` node with `pipeline_type = 1536_cascade`, the process always crashes with a `Fatal Python error: Aborted` after the HR sampling completes (100%).

The crash always occurs at the same location:
```
File "trellis2/modules/sparse/spatial/spatial2channel.py", line 77 in forward
```

The crash is **not** related to VRAM — the GPU has 24GB and VRAM usage stays around 50% throughout the entire process.

### Root Cause
In `spatial2channel.py` line 77, the computed index `idx * self.factor ** DIM + subidx` exceeds the bounds of the `new_feats` tensor when the sparse structure generated by `1536_cascade` is large enough. This causes an out-of-bounds memory access and Python aborts.

```python
# Line 76-77 - problematic code
new_feats = torch.zeros(new_coords.shape[0] * self.factor ** DIM, x.feats.shape[1], device=x.feats.device, dtype=x.feats.dtype)
new_feats[idx * self.factor ** DIM + subidx] = x.feats  # <-- index out of bounds
```

### Workaround
Adding `.clamp()` prevents the crash but produces an incomplete/broken mesh:
```python
new_feats = torch.zeros(new_coords.shape[0] * self.factor ** DIM, x.feats.shape[1], device=x.feats.device, dtype=x.feats.dtype)
flat_idx = (idx * self.factor ** DIM + subidx).clamp(0, new_feats.shape[0] - 1)
new_feats[flat_idx] = x.feats
```

### Steps to Reproduce
1. Use `Trellis2 - Mesh With Voxel Multi-View Generator` node
2. Set `pipeline_type = 1536_cascade`
3. Set `sparse_structure_resolution = 32`
4. Run with any input image
5. Process completes HR sampling (50/50) then crashes

### Expected Behavior
Mesh should be generated successfully, same as with `1024_cascade`.

### Notes
- The bug does **not** occur with `1024_cascade` pipeline
- The bug occurs consistently regardless of attention backend (flash_attn, sage_attn)
- The bug occurs regardless of `--highvram` / `--normalvram` flags
- VRAM is not the issue — confirmed with monitoring tools

Test input images that reproduce the crash (front and back view):
<img width="1536" height="1536" alt="Image" src="https://github.com/user-attachments/assets/0b2c320a-91a9-4a34-ab48-6a75ee8e1318" />
<img width="1536" height="1536" alt="Image" src="https://github.com/user-attachments/assets/eb07f007-f21a-44da-87d8-87a0b0321d7a" />

**Comment by PRSRVFRSHNSS** [comment]
Did you ever get this ironed out beyond the workaround that outputs a broken mesh?
